README.md

```diff
@@ -57,8 +57,15 @@ The steps are:
     ```
 
 3. Build and run the Docker containers. This might take a while.
+    
```

Comment:
Hi @dubesar ,
I don't think we need these changes as you can  add docker user in root user group. For more details please read here: https://docs.docker.com/install/linux/linux-postinstall/